src/controllers/tasks/confirm.email.address.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../../types/template.paths";
+import { urlUtils } from "../../utils/url";
+import { PROVIDE_EMAIL_ADDRESS_PATH, TASK_LIST_PATH } from "../../types/page.urls";
+import { Session } from "@companieshouse/node-session-handler";
+import { SECTIONS } from "../../utils/constants";
+import { sendUpdate } from "../../utils/update.confirmation.statement.submission";
+import { SectionStatus } from "@companieshouse/api-sdk-node/dist/services/confirmation-statement";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const session = req.session as Session;
+    const backLinkUrl = urlUtils.getUrlToPath(PROVIDE_EMAIL_ADDRESS_PATH, req);
+    const emailAddress = session.getExtraData("entered-email-address");
```

Comment:
Is the entered-email-address session data already set? I don't see it in this PR, is there a change to provide.email.address.controller.ts too?

Reply:
Resolved: It got wiped out and missed during a manual conflict merge when registered.email.address.controller was renamed. Reinstated now.